dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ranger/RangerConnector.java

```diff
@@ -89,11 +97,31 @@ public void addTasksTo(
   @Nonnull
   @Override
   public Handle open(@Nonnull ConnectorArguments arguments) throws Exception {
-    // TODO: handle SSL or Kerberos.
-    URI apiUrl = URI.create("http://" + arguments.getHostOrDefault() + ":" + arguments.getPort());
+    URIBuilder uriBuilder =
+        new URIBuilder().setPort(arguments.getPort()).setHost(arguments.getHostOrDefault());
+    if (Objects.equals(arguments.getRangerScheme(), "https")) {
```

Comment:
If I understand correctly, if a user passes an invalid schema (let's say hello-world), it'll end up using just http.

I would propose to `uriBuilder.setScheme(arguments.getRangerScheme())` and delegate validation to http client.